package.json

```diff
@@ -63,7 +51,7 @@
     "tsconfig-paths": "^3.12.0",
     "ttypescript": "^1.5.12",
     "typemoq": "^2.1.0",
-    "typescript": "4.3.5",
+    "typescript": "^4.5.5",
```

Comment:
Jest-Preset-Angular was pulling in angular/compiler-clie@13.3.0 and complaining, but you're right it'd be better to install with legacy deps since jest-preset-angular claims it needs >10.0.0.

Should be fixed